Common/RotationalEncoder.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <cmath>
+#include <cstdint>
+
+#include "utility/status.hpp"
+
+namespace sjsu
+{
+class RotationalEncoder
+{
+ public:
+  virtual Returns<void> Initialize()            = 0;
+  virtual Returns<units::degrees> GetRotation() = 0;
```

Comment:
This line will not compile. Test this out in SJSU-Dev2 if you need to but you will see that it does not work.